src/server/handlers/launchpad.js

```diff
@@ -726,8 +769,9 @@ export const requestSnapBuilds = async (req, res) => {
     const { owner, name } = await checkAdminPermissions(
       req.session, req.body.repository_url
     );
+    const reason = req.body.reason || BUILD_TRIGGERED_MANUALLY;
     const snap = await internalFindSnap(req.body.repository_url);
-    const builds = await internalRequestSnapBuilds(snap, owner, name);
+    const builds = await internalRequestSnapBuilds(snap, owner, name, reason);
     return res.status(201).send({
       status: 'success',
       payload: {
```

Comment:
Currently `requestSnapBuilds` payload only contains builds.
It would be good if it would return `build_annotations` as well, similar to `getSnapBuilds`, so once new builds are requested we can immediately show them in UI with annotations (without refetching them from server).

Reply:
Good point, I will include the just created build_annotations.